### PR TITLE
feat: add experienceLevel column to user table

### PR DESCRIPTION
**Pull request by kopancek**
Need to add this column to the user table, so that it can be used later on in the User profile page and on signup pages.

Related issue: 
- https://dailydotdev.atlassian.net/browse/MI-314

Related UI PR: 
- https://github.com/dailydotdev/apps/pull/3037

Related Heimdall PR:
- https://github.com/dailydotdev/heimdall/pull/94

### CODE REVIEW

**Review by rebelchris — COMMENTED** [review]
Don't forget to also add this logic to `/p/newUser` endpoint that heimdall uses.

File: src/schema/users.ts [hunk 1]
```diff
@@ -88,6 +88,7 @@ export interface GQLUpdateUserInput {
   acceptedMarketing?: boolean;
   notificationEmail?: boolean;
   infoConfirmed?: boolean;
+  experienceLevel?: string;
```
[the remark this answers is not in the document]
Comment (kopancek) [reply src/schema/users.ts:91]: This should be done now.

**Comment by kopancek** [comment]
> Don't forget to also add this logic to `/p/newUser` endpoint that heimdall uses.

Heimdall PR merged and the endpoint was updated.

**Review by capJavert — COMMENTED** [review]
@kopancek can you add a tests in `__tests__/private` to confirm traits is being passed over to new users in `/newUser` endpoint. Other then that looks good.

**Review by rebelchris — APPROVED** [review]

**Comment by kopancek** [comment]
> @kopancek can you add a tests in `__tests__/private` to confirm traits is being passed over to new users in `/newUser` endpoint. Other then that looks good.

I added a test to `private.ts` that checks if `experienceLevel` is assigned correctly on the `/newUser` endpoint. Heimdall converts traits to fields that are being sent over to the `/newUser` endpoint, so there's no way to test that.

**Review by capJavert — APPROVED** [review]
Nice 👍

**Pull request merged**